packages/buttons/src/elements/ButtonGroup.tsx

```diff
@@ -13,7 +13,13 @@ import { StyledButtonGroup } from '../styled';
 import { ButtonGroupContext } from '../utils/useButtonGroupContext';
 
 export interface IButtonGroupProps extends HTMLAttributes<HTMLDivElement> {
+  /** Set the selected item */
```

Comment:
Follow grammar guidelines and maybe be more descriptive, something like:

> Sets the selected item within the group

Reply:
Fixed in latest push.